### DISCUSSION TITLE
Add "colour blind" mode

### DISCUSSION THREAD

**Discussion by akmal1223233** [ppy/osu#17058] [Ideas]
would like to see this
hopefully this stuff isn't useless

**Comment by bdach** [comment]
Colour matters in many more places than just taiko. So this would be a game-wide accessibility option rather than something taiko-specific.

**Reply by peppy** [reply]
I believe other games do this using shader-level changes. It is probably something we can look into.

**Comment by peppy** [comment]
@akmal1223233 can you tell us which kind of colour blindness you suffer from?

**Reply by akmal1223233** [reply]
nothing
